python/tests/test-export-classify.sh

```diff
@@ -0,0 +1,86 @@
+BASELINE_DIR=${HOME}/dev/work/baseline
+CLASSIFY_MODEL=sst2-25195.zip
+TEST_FILE=stsa.binary.test
+TEST_LOAD=$TEST_FILE.load
+TEST_SERVE=$TEST_FILE.serve
+TEST_SERVE_PREPROC=$TEST_FILE.serve_preproc
+SLEEP=5
+
+docker stop tfserving
```

Comment:
Is this an alias on your system? I get this error

```
Error response from daemon: No such container: tfserving
Error: No such container: tfserving
```

The docs says the image is called tensorflow/serving

Reply:
ignore the error.